Makefile

```diff
@@ -1,3 +1,11 @@
+BUILDTAGS ?= \
+	$(shell hack/btrfs_installed_tag.sh) \
+	$(shell hack/btrfs_tag.sh) \
+	exclude_graphdriver_devicemapper \
+	exclude_graphdriver_btrfs
+
+GOFLAGS=-mod=vendor
```

Comment:
are we going to use vendor?

Reply:
RHEL btrfs problems, I'm going to leave commented in case of needed